packages/webchat/src/main.tsx

```diff
@@ -224,6 +224,9 @@ class Web extends React.Component<MainProps> {
     if (this.props.currentConversation?.userId !== event.authorId) {
       trackMessage('received')
       postMessageToParent('MESSAGE.RECEIVED', event, this.props.config!.chatId)
+      if (event.id) {
```

Comment:
Make sure to add a comment that explains this change, its use case, why it's there, and everything!